zeebe/stream-platform/src/test/java/io/camunda/zeebe/stream/impl/RecordBatchTest.java

```diff
@@ -159,7 +159,7 @@ void shouldOnlyAddUntilMaxBatchSizeIsReached() {
     assertThat(either.isLeft()).isTrue();
     assertThat(either.getLeft())
         .hasMessageContaining("Can't append entry")
-        .hasMessageContaining("[ currentBatchEntryCount: 1, currentBatchSize: 416]");
+        .hasMessageContaining("[ currentBatchEntryCount: 1, currentBatchSize: 422]");
```

Comment:
Dear Engine Reviewer, I just took a leap here to make this test work. I don't know if there is some defined limit that would be breached with this change. 💭

Reply:
This is okay, this test covers the rejection functionality, so the predicate is "simulated" on line 150 above. The (production) limit can be variable and is calculated [here](https://github.com/camunda/camunda/blob/17e57f88b7be1c36642ad44b40a058e79843a46a/zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/Sequencer.java#L67-L73). We have different tests for that 🙂